gps.orogen

```diff
@@ -5,71 +5,101 @@ using_library 'mb500'
 using_library 'libgps'
 
 import_types_from 'base'
+import_types_from 'gps_base'
 # We get our interface types from the mb500 library even for the gpsd driver
-import_types_from 'gps_types.hh'
 import_types_from 'mb500_types.hh'
 
+# Export deprecated types for backward compatibility
+import_types_from 'gps_types.hh'
+typekit do
+    export_types '/gps/Solution',
```

Comment:
There isn't much backward compatibility since the task interfaces change anyways.

If you could try to typedef the gps types from the gps_base types, it may be enough to allow components that would use gps/* types to transition to gps_base/

Reply:
I removed the export of the old types and defined typedefs in the library rock-drivers/drivers-mb500#3.